action.yml

```diff
@@ -56,30 +57,31 @@ runs:
         # If the output is not empty, there are changes. Commit them
         if [ -n "$git_changes" ]; then
             echo "Changes to workflows found!"
+            REPOSITORY_NAME="$(echo "$GITHUB_REPOSITORY" | awk -F / '{print $2}' | sed -e "s/:refs//")"
             BRANCH_NAME="${REPOSITORY_NAME}-github-actions-self-update"
             echo "Using branch: $BRANCH_NAME"
             git checkout -b "$BRANCH_NAME"
-
+        
```

Comment:
Why this whitespace on the blank lines? Is this just the script importer? Not a big deal if that is the case.

Reply:
Yes, because kojo/erb indents the whole file including blank lines when imported.